main.py

```diff
@@ -23,14 +30,39 @@ def setup(self):
         log.info("Setup from of_multi_table")
 
     def execute(self):
-        """Run after the setup method execution.
+        """Execute once when the napp is running."""
+        log.info("Executing of_multi_table")
 
-        You can also use this method in loop mode if you add to the above setup
-        method a line like the following example:
+    @rest("/v1/pipeline", methods=["POST"])
+    @validate_openapi(spec)
+    def add_pipeline(self, data):
+        """Add pipeline"""
+        return jsonify("Operation successful"), 200
```

Comment:
Typically HTTP creation returns 201, could you adapt this?

Reply:
Changed